README.md

```diff
@@ -88,113 +88,155 @@ When using Coverage.jl locally, over time a lot of `.cov` files can accumulate.
-2. You must be using `Julia 0.3` or higher, which added the `--code-coverage` command line argument.
-3. Use the command line option when you run your tests
-  * Either with something like `julia --code-coverage test/runtests.jl`, or
-  * with something like  `julia -e 'Pkg.test("MyPkg", coverage=true)'`
-4. Add the following to the end of your `.travis.yml` or `.appveyor.yml` file. This line downloads this package, collects the per-file coverage data, then bundles it up and submits to Coveralls. Coverage.jl assumes that the working directory is the package directory, so it changes to that first (so don't forget to replace `MyPkg` with your package's name!
-  * On Travis CI:
-  ```yml
-  after_success:
-  - julia -e 'cd(Pkg.dir("MyPkg")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
-  ```
-  * On AppVeyor:
-  ```yml
-  after_test:
-  - C:\projects\julia\bin\julia -e "cd(Pkg.dir(\"MyPkg\")); Pkg.add(\"Coverage\"); using Coverage; Coveralls.submit(process_folder())"
-  ```
-
-
-## Julia packages using Coverage.jl
+1. Enable [Coveralls.io](https://coveralls.io) for your repository. If it is
+   public on GitHub and you are using TravisCI, this is all you need to do. If
+   you are using AppVeyor, you need to add a secure environment variable
+   called `COVERALLS_TOKEN` to your `.appveyor.yml` (see
+   [here](https://www.appveyor.com/docs/build-configuration/#secure-variables)).
+   Your repo token can be found in your Coveralls repo settings. If neither of
+   these are true, please submit an issue, and we can work on adding
+   additional functionality for your use case.
+
+2. Activate the `--code-coverage` command line option when you run your tests
+   - Either with something like `julia --code-coverage test/runtests.jl`, or
+   - with something like  `julia -e 'Pkg.test("MyPkg", coverage=true)'`
+
+3. Configure your CI service to upload coverage data:
+
+   - If you are using Travis with `language: julia`, simply add `coveralls: true`
+     to your `.travis.yml`.
+
+   - You can also add the following to the end of your `.travis.yml`. This
+     line downloads this package, collects the per-file coverage data, then
+     bundles it up and submits to Coveralls. Coverage.jl assumes that the
+     working directory is the package directory, so it changes to that first
+     (so don't forget to replace `MyPkg` with your package's name!
+
+   - On Travis CI:
+
+       ```yml
+       after_success:
+       - julia -e 'cd(Pkg.dir("MyPkg")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
```

Comment:
`cd(Pkg.dir("..."))` is for Julia 0.6 and prior; for 0.7 and later, that step should be replaced with `using Pkg`.

Reply:
This code is indeed missing a `using Pkg`, thanks for catching it. The `cd` is indeed not needed for packages using a `Project.toml`. For older packages, I think it might be needed -- but since of course this README is only interesting for new packages, and those should all be using `Project.toml`, So overall, I agree, and pushed a corresponding change. Thank you!